<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1000</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Salvador's Render</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(51, 51, 51);</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="enabled">
    <bool>true</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>679</x>
     <y>0</y>
     <width>321</width>
     <height>601</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(40, 40, 40);</string>
   </property>
   <widget class="QScrollArea" name="scrollArea">
    <property name="geometry">
     <rect>
      <x>9</x>
      <y>0</y>
      <width>321</width>
      <height>601</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color: rgb(51, 51, 51);</string>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>319</width>
       <height>599</height>
      </rect>
     </property>
     <widget class="Line" name="line">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>110</y>
        <width>291</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>10</y>
        <width>291</width>
        <height>93</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="layout_Transfrorm">
       <item>
        <widget class="QLabel" name="label">
         <property name="styleSheet">
          <string notr="true">font: 700 9pt &quot;Arial&quot;;</string>
         </property>
         <property name="text">
          <string>Transform</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="styleSheet">
              <string notr="true">font: 700 8pt &quot;Arial&quot;;</string>
             </property>
             <property name="text">
              <string>Position</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout">
               <item>
                <widget class="QLabel" name="label_3">
                 <property name="styleSheet">
                  <string notr="true">font: 700 8pt &quot;Arial&quot;;</string>
                 </property>
                 <property name="text">
                  <string>X</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lEd_posX">
                 <property name="styleSheet">
                  <string notr="true">font: 700 8pt &quot;Arial&quot;;</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignmentFlag::AlignBottom|Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft</set>
                 </property>
                 <property name="placeholderText">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_2">
               <item>
                <widget class="QLabel" name="label_4">
                 <property name="styleSheet">
                  <string notr="true">font: 700 8pt &quot;Arial&quot;;</string>
                 </property>
                 <property name="text">
                  <string>Y</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lEd_posY">
                 <property name="styleSheet">
                  <string notr="true">font: 700 8pt &quot;Arial&quot;;</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignmentFlag::AlignBottom|Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft</set>
                 </property>
                 <property name="placeholderText">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_3">
               <item>
                <widget class="QLabel" name="label_5">
                 <property name="styleSheet">
                  <string notr="true">font: 700 8pt &quot;Arial&quot;;</string>
                 </property>
                 <property name="text">
                  <string>Z</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lEd_posZ">
                 <property name="styleSheet">
                  <string notr="true">font: 700 8pt &quot;Arial&quot;;</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignmentFlag::AlignBottom|Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft</set>
                 </property>
                 <property name="placeholderText">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_6">
           <item>
            <widget class="QLabel" name="label_6">
             <property name="styleSheet">
              <string notr="true">font: 700 8pt &quot;Arial&quot;;</string>
             </property>
             <property name="text">
              <string>Rotation</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_7">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_8">
               <item>
                <widget class="QLabel" name="label_7">
                 <property name="styleSheet">
                  <string notr="true">font: 700 8pt &quot;Arial&quot;;</string>
                 </property>
                 <property name="text">
                  <string>X</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lEd_rotX">
                 <property name="styleSheet">
                  <string notr="true">font: 700 8pt &quot;Arial&quot;;</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignmentFlag::AlignBottom|Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft</set>
                 </property>
                 <property name="placeholderText">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_9">
               <item>
                <widget class="QLabel" name="label_8">
                 <property name="styleSheet">
                  <string notr="true">font: 700 8pt &quot;Arial&quot;;</string>
                 </property>
                 <property name="text">
                  <string>Y</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lEd_rotY">
                 <property name="styleSheet">
                  <string notr="true">font: 700 8pt &quot;Arial&quot;;</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignmentFlag::AlignBottom|Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft</set>
                 </property>
                 <property name="placeholderText">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_10">
               <item>
                <widget class="QLabel" name="label_9">
                 <property name="styleSheet">
                  <string notr="true">font: 700 8pt &quot;Arial&quot;;</string>
                 </property>
                 <property name="text">
                  <string>Z</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLineEdit" name="lEd_rotZ">
                 <property name="styleSheet">
                  <string notr="true">font: 700 8pt &quot;Arial&quot;;</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignmentFlag::AlignBottom|Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft</set>
                 </property>
                 <property name="placeholderText">
                  <string/>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="Line" name="line_2">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>300</y>
        <width>291</width>
        <height>16</height>
       </rect>
      </property>
      <property name="orientation">
       <enum>Qt::Orientation::Horizontal</enum>
      </property>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>320</y>
        <width>291</width>
        <height>120</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_9">
       <property name="leftMargin">
        <number>0</number>
       </property>
       <item>
        <widget class="QLabel" name="label_14">
         <property name="styleSheet">
          <string notr="true">font: 700 9pt &quot;Arial&quot;;</string>
         </property>
         <property name="text">
          <string>Object</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_8">
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_7">
           <item>
            <widget class="QLabel" name="label_19">
             <property name="styleSheet">
              <string notr="true">font: 700 8pt &quot;Arial&quot;;</string>
             </property>
             <property name="text">
              <string>Type</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="comboBox">
             <item>
              <property name="text">
               <string>Cube</string>
              </property>
             </item>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_15">
           <item>
            <widget class="QLabel" name="label_18">
             <property name="styleSheet">
              <string notr="true">font: 700 8pt &quot;Arial&quot;;</string>
             </property>
             <property name="text">
              <string>Color</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_14" stretch="8,2">
             <property name="leftMargin">
              <number>30</number>
             </property>
             <item>
              <widget class="QLabel" name="lbl_colorShow">
               <property name="styleSheet">
                <string notr="true">font: 700 8pt &quot;Arial&quot;;
background-color: rgb(255, 0, 0);</string>
               </property>
               <property name="frameShape">
                <enum>QFrame::Shape::Box</enum>
               </property>
               <property name="frameShadow">
                <enum>QFrame::Shadow::Sunken</enum>
               </property>
               <property name="text">
                <string/>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QPushButton" name="btn_ColorPicker">
               <property name="text">
                <string>...</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="layoutWidget">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>130</y>
        <width>291</width>
        <height>161</height>
       </rect>
      </property>
      <layout class="QVBoxLayout" name="layout_Camera">
       <item>
        <widget class="QLabel" name="label_10">
         <property name="styleSheet">
          <string notr="true">font: 700 9pt &quot;Arial&quot;;</string>
         </property>
         <property name="text">
          <string>Camera</string>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QVBoxLayout" name="verticalLayout_6">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_11">
           <item>
            <widget class="QLabel" name="label_11">
             <property name="styleSheet">
              <string notr="true">font: 700 8pt &quot;Arial&quot;;</string>
             </property>
             <property name="text">
              <string>View</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QDial" name="dial_Phi">
             <property name="maximum">
              <number>360</number>
             </property>
             <property name="value">
              <number>30</number>
             </property>
             <property name="invertedAppearance">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="label_12">
             <property name="styleSheet">
              <string notr="true">font: 8pt &quot;Arial&quot;;</string>
             </property>
             <property name="text">
              <string>30</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="verticalLayout_4">
           <item>
            <widget class="QLabel" name="label_13">
             <property name="styleSheet">
              <string notr="true">font: 700 8pt &quot;Arial&quot;;</string>
             </property>
             <property name="text">
              <string>Position</string>
             </property>
            </widget>
           </item>
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_12">
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_3">
               <item>
                <widget class="QLabel" name="label_15">
                 <property name="styleSheet">
                  <string notr="true">font: 8pt &quot;Arial&quot;;</string>
                 </property>
                 <property name="text">
                  <string>Horizontal</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="label_17">
                 <property name="styleSheet">
                  <string notr="true">font: 8pt &quot;Arial&quot;;</string>
                 </property>
                 <property name="text">
                  <string>Vertical</string>
                 </property>
                 <property name="alignment">
                  <set>Qt::AlignmentFlag::AlignLeading|Qt::AlignmentFlag::AlignLeft|Qt::AlignmentFlag::AlignVCenter</set>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
             <item>
              <layout class="QVBoxLayout" name="verticalLayout_2">
               <item>
                <widget class="QSlider" name="slider_CameraX">
                 <property name="maximum">
                  <number>300</number>
                 </property>
                 <property name="value">
                  <number>100</number>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Orientation::Horizontal</enum>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QSlider" name="slider_CameraV">
                 <property name="maximum">
                  <number>300</number>
                 </property>
                 <property name="value">
                  <number>100</number>
                 </property>
                 <property name="orientation">
                  <enum>Qt::Orientation::Horizontal</enum>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </widget>
    </widget>
   </widget>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>2</width>
     <height>2</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_5"/>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
